cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -385,15 +388,29 @@ func (s *CMServer) Run(_ []string) error {
 		}
 	}
 
+	volumePlugins := ProbeRecyclableVolumePlugins(s.VolumeConfigFlags)
+	provisioner, provisionerExpected, err := NewVolumeProvisioner(volumePlugins, cloud, s.VolumeConfigFlags)
+	if provisionerExpected && err != nil {
```

Comment:
I'd prefer that you just return a provisioner and an error. If the error is non-nil, then you exit.